### ISSUE TITLE
protobuf 2.6.0 breaks pycaffe on OS X

### ISSUE THREAD

**Issue by shelhamer** [BVLC/caffe#1202]
Stay at 2.5.0 or downgrade. Otherwise `import caffe` gives

```
/Users/shelhamer/python/caffe/proto/caffe_pb2.py in <module>()
      8 from google.protobuf import message as _message
      9 from google.protobuf import reflection as _reflection
---> 10 from google.protobuf import symbol_database as _symbol_database
     11 from google.protobuf import descriptor_pb2
     12 # @@protoc_insertion_point(imports)

ImportError: cannot import name symbol_database
```

**Comment by anguyen8** [comment]
I'm having the same problem here with protobuf 2.5.0, latest caffe and CUDA 6.5. 
I'd appreciate if someone has a solution.

**Comment by dionyziz** [comment]
I'm getting the exact same error when I try to run https://github.com/google/grr on travis. Perhaps it's something we're doing wrong with protobufs instead of a project issue?

If you find a solution, please let us know. Thanks!

**Comment by anguyen8** [comment]
@dionyziz : I was using Python Anaconda and installed Protobuf python lib on it. My problem was fixed when I switched to not using Anaconda.

**Comment by dionyziz** [comment]
This was a protobuf version error for me. I solved it by switching from protobuf==2.5.0 to protobuf==2.6.0.

**Comment by shelhamer** [comment]
Not sure what the problem was back then, but protobuf 2.6.1 works on OS X -- closing.
